src/main/kotlin/org/jetbrains/plugins/bsp/magicmetamodel/impl/workspacemodel/WorkspaceModelEntities.kt

```diff
@@ -53,7 +53,9 @@ public data class Library(
   val iJars: List<String> = listOf(),
   val sourceJars: List<String> = listOf(),
   val classJars: List<String> = listOf(),
-) : WorkspaceModelEntity(), ResourceRootEntity {
+  val goImportPath: String? = "",
```

Comment:
powinniśmy to dodać również do LibraryState - czyli także do funkcji Library.toState i LibraryState.fromState - tu też muszę abramsowi przypomnieć, że nie są dodane `iJars`